pkg/apis/duck/v1alpha1/channelable_combined_types.go

```diff
@@ -80,9 +80,9 @@ type ChannelableCombinedStatus struct {
 	// SubscribableTypeStatus is the v1alpha1 part of the Subscribers status
 	SubscribableTypeStatus `json:",inline"`
 	// SubscribableStatus is the v1beta1 part of the Subscribers status.
-	eventingduckv1beta1.SubscribableStatus `json:",inline"`
+	SubscribableStatusv1beta1 eventingduckv1beta1.SubscribableStatus `json:",inline"`
```

Comment:
Just to make sure I understand. We should be dropping the duck/v1alpha1, since we're dropping the support for v1alpha1 API shapes as per:
https://knative.dev/community/contributing/mechanics/release-versioning-principles/

They were not served in the .16, so we should not be using them here. Are we just leaving the pkg in v1alpha1 as is to make the changes smaller? I guess what I'm asking is, I feel like we should be dropping the duck/v1alpha1/channelable* related things, because v1alpha1 API is no longer supported, so would make more sense to only have a v1beta1 and v1 only. Or, am I missing something (still catching up after vacation :) )

Reply:
that's good to hear. dropping v1alpha1 would make things easier.

Reply:
thanks for commenting @vaikas. Dropping v1alpha support will make this simpler. I was not aware that was the plan. I will update this PR (and likely revert some of the code in https://github.com/knative/eventing/pull/3803 as it should not be necessary, and ping you to review once that is ready.
Thanks!

Reply:
awesome sauce, thanks much for doing this!

Reply:
one extra thing to keep in mind... if we drop v1alpha1 now, I think eventing-contrib won't be able to update-deps until we move natss out of there or we fix it..

Reply:
Issue to migrate NATSSChannel to v1 Channelable: https://github.com/knative/eventing-contrib/issues/1485
Issue to move NATSSChannel to a separate repo: https://github.com/knative/eventing-contrib/issues/1487

Reply:
https://github.com/knative/eventing/pull/3871 is now ready for review. Not sure if it needs to be put on hold until all deps are also updated. Please put on hold if needed. Will be closing this one.